src/lang-pt_BR.js

```diff
@@ -0,0 +1,517 @@
+    'dark-cyan-3': 'ciano escuro 3',
+    'light-cornflower-blue-3': 'azul flor do milho claro 3',
+    'light-cornflower-blue-2': 'azul flor do milho claro 2',
+    'light-cornflower-blue-1': 'azul flor do milho claro 1',
+    'dark-cornflower-blue-1': 'azul flor do milho escuro 1',
+    'dark-cornflower-blue-2': 'azul flor do milho escuro 2',
+    'dark-cornflower-blue-3': 'azul flor do milho escuro 3',
+    'light-blue-3': 'azul claro 3',
+    'light-blue-2': 'azul claro 2',
+    'light-blue-1': 'azul claro 1',
+    'dark-blue-1': 'azul escuro 1',
+    'dark-blue-2': 'azul escuro 2',
+    'dark-blue-3': 'azul escuro 3',
+    'light-purple-3': 'roxo claro 3',
+    'light-purple-2': 'roxo claro 2',
+    'light-purple-1': 'roxo claro 1',
+    'dark-purple-1': 'roxo escuro 1',
+    'dark-purple-2': 'roxo escuro 2',
+    'dark-purple-3': 'roxo escuro 3',
+    'light-magenta-3': 'magenta claro 3',
+    'light-magenta-2': 'magenta claro 2',
+    'light-magenta-1': 'magenta claro 1',
+    'dark-magenta-1': 'magenta escuro 1',
+    'dark-magenta-2': 'magenta escuro 2',
+    'dark-magenta-3': 'magenta escuro 3',
+    Shapes: 'Formas',
+    disableAllChat: 'Desabilitar bate-papo privado',
+    enableAllChat: 'Habilitar bate-papo privado',
+    disableAllGroupChat: 'Desabilitar bate-papo comum',
+    enableAllGroupChat: 'Habilitar bate-papo comum',
+    disableAllRaisehand: 'Desabilitar levantar mão',
+    enableAllRaisehand: 'Habilitar levantar mão',
+    disableAllAskQuestion: 'Desabilitar levantar questão',
+    enableAllAskQuestion: 'Habilitar levantar questão',
+    disableAllQaMarkNotes: 'Desabilitar marcações e anotações',
+    enableAllQaMarkNotes: 'Habilitar marcações e anotações',
+    disableAllUserlist: 'Desabilitar lista de usuários',
+    enableAllUserlist: 'Habilitar lista de usuários',
+    Audio: 'áudio',
+    Video: 'vídeo',
+    Chat: 'bate-papo privado',
+    GroupChat: 'bate-papo comum',
+    RaiseHand: 'levantar mão',
+    UserList: 'lista de usuários',
+    uploading: 'Processando...',
+    userListHeader: 'Usuários ',
+    commonChatHeader: 'Sala de Bate-papo',
+    teacherVideoHeader: 'Vídeo do professor',
+    askQuestionHeader: 'Levantar Questão',
+    questionLable: 'Questão',
+    answerLable: 'Resposta',
+    commentLable: 'Comentário',
+    markAnswer: 'Marcar como Resposta',
+    markNotes: 'Marcações e Anotações',
+    askQuestionTimeExceed: 'Incapaz de editar/deletar pois você já excedeu o tempo limite',
+    upvoted: 'Incapaz de editar/deletar pois um voto a favor já foi adicionado',
+    enterText: 'Inserir o texto antes de postar',
+    more: '...Mais',
+    less: 'Menos',
+    askQuestion: 'Levantar questão',
+    MarksAndNotes: 'Marcações e Anotações',
+    markAnswerUnmark: 'Você precisa primeiro desmarcar a resposta já marcada',
+    /* For Chrome */
+    PermissionDeniedErrorExt: '<div class="errorMsg"> Acesso à câmera foi bloqueado. Para providenciar acesso à Webcam, favor seguir os procedimentos abaixo <br />'
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [This line has a length of 164. Maximum allowed is 120.](https://app.codacy.com/gh/congrea/virtualclass/pullRequest?prid=5482325)